api/streak.js

```diff
@@ -0,0 +1,87 @@
+import renderStreaksCard from "../src/cards/streak-card.js";
+import { blacklist } from "../src/common/blacklist.js";
+import {
+  clampValue,
+  CONSTANTS,
+  parseArray,
+  parseBoolean,
+  renderError,
+} from "../src/common/utils.js";
+
+import { isLocaleAvailable } from "../src/translations.js";
+import { streakFetcher } from "../src/fetchers/streak-fetcher.js";
+import axios from "axios";
+export default async (req, res) => {
+  const {
+    username,
+    hide,
+    hide_border,
```

Comment:
I think it would be nice if the card also included icons 🤔.

Reply:
@yaten2302 O damn, your right that's a bit hard. My initial thoughts were you had the same options as https://github.com/DenverCoder1/github-readme-streak-stats.

- **Total contributions:** https://fontawesome.com/icons/cube?f=classic&s=light (a box - inspired by GitHub's activity view).
- **Longest streak:** https://fontawesome.com/icons/fire?f=classic&s=light (normal fire).
- **Current streak:** https://fontawesome.com/icons/fire?f=classic&s=solid (solid fire).

I, however, now see that you have a lot more options. Can you explain why users would want to show these other options 🤔?